lib/robot-simulator.js

```
@@ -0,0 +1,281 @@
+        if (this.battery >= amount) {
+            this.battery -= amount;
+            return true;
+        }
+        return false;
+    };
+    MarsRobot.prototype.executeCommand = function (command) {
+        switch (command) {
+            case "F":
+                if (!this.consumeBattery(3))
+                    return false;
+                var nextPosF = this.getNextPosition(true);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)